presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,6 +171,55 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules govern who is considered an owner of a schema. The user is granted
+the privileges based on the first matching rule read from top to bottom. If no
+rule matches, the user is not considered an owner of the schema. Each rule is
+composed of the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema names. Defaults to ``.*``.
+* ``owner`` (required): boolean indicating whether the user is to be considered
+  an owner of the schema. Defaults to ``false``.
+
+For example, to provide ownership of all schemas to user ``admin``, treat all
+users as owners of ``default`` schema and prevent user ``guest`` from ownership
+of any schema, you can use the following rules:
+
+.. literalinclude:: schema-access.json
+    :language: json
+
+.. _table_rules:
+
+Table Rules
```

Comment:
With the way the table rules are actually evaluated in the built-in SAC means that by if no table rules are provided all users are assumed to have all privileges (documented now in this PR).

In addition to this all tables in information_schema are always allowed regardless of the rules in the config file. Should this be documented?

Reply:
information schema behavior should be documented

Reply:
Documented within a note in the section (similar to catalog rules).